### ISSUE TITLE
I can't install it manually using "python.exe setup.py install"

### ISSUE THREAD

**Issue by Ademir-Francisco-da-Silva** [riklaunim/django-ckeditor#49]
It doesn't create a folder like this "django_ckeditor_updated-4.4.0-py2.7.egg" and even theirs files ( *.py, *.pyc and etc ... )

Further information ...:
Python 2.7.8
Django 1.6.5
Windows Vista_64 SP2

**Comment by riklaunim** [comment]
I did a test and it installs for Python 3.4 on Win7. Maybe it installs in your case (different path etc.)?

**Comment by SalahAdDin** [comment]
Is possible install with python3.4?

**Comment by riklaunim** [comment]
Yes, Python 3.3 or higher or Python 2.7 are supported. It should install or throw some sort of an exception. After "python setup.py install" try importing "ckeditor" - if it's installed it will import and throw exceptions about django settings not set and so on.

**Comment by Ademir-Francisco-da-Silva** [comment]
Ok. Let's go so ...

My machine's path is ...: 
C:\Python27\Lib\site-packages\django_ckeditor_updated-4.2.8-py2.7.egg, 
for example, and
works well, but when I try to install that file from ( 
django-ckeditor-updated-4.4.0.tar.gz ) ...

https://pypi.python.org/packages/source/d/django-ckeditor-updated/django-ckeditor-updated-4.4.0.tar.gz#md5=19829ecc11e2735c2f48f8a840aea310

using  python.exe setup.py install  it fails. I can't see the directory 
created like this one 
C:\Python27\Lib\site-packages\django_ckeditor_updated-4.4.0-py2.7.egg in 
my machine's path, this is the problem ...

Thanks for your help ..

Ademir

PS .: I have tried to "import ckeditor" via IPython 2.2.0 and nothing 
happend because the installer wasn't work properly ...

On 24/08/2014 15:18, riklaunim wrote:

> I did a test and it installs for Python 3.4 on Win7. Maybe it installs 
> in your case (different path etc.)?
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/riklaunim/django-ckeditor/issues/49#issuecomment-53201995.

## 

Ademir Francisco da Silva
Skype  ...: Ademir_Francisco_da_Silva
[ CMC ] God will do me justice
